### ISSUE TITLE
Change cookie to a local storage object for sake of expediency of development

### ISSUE THREAD

**Issue by dtolmie** [Trivia-Crossroads/triviacrossroads#86]

**Comment by dtolmie** [comment]
I think we should forget about the cookie stuff, unless we have everything else done and would rather create cookie code instead of find celebratory beverages. I'm feeling the cookie less and less, since getting the cookies going will then necessitate a whole new page. Let's talk about this in the AM.

**Comment by dtolmie** [comment]
Okay. I caved. I added the code to write a super cookie to localStorage. The cookie is getting written to localStorate as 'triviaCrossroadsUUID = Firebase_UUID'. This allows us to pull the specific user from the Firebase DB. This will make it pretty simple to make the profile page, or another page, show a user's data if the cookie exists. We could even make the profile form for appear prepopulated and allow them to update their details.
